citi-onboarding/DADe-onepage
# feat(Feature/create-realizations-view): Created realizations view (desktop and mobile)

## What I did

- Created Realizations view fully responsive (desktop and mobile)
- Made Realizations connection to backend
- Changed backend inputs to fit new mockup settings
- Changed notion footer color to new mockup configuration
- Remove link from email on contacts view

| Mockup (Desktop) | Real (Desktop) |
|--------|------|
| ![Realizations mockup desktop](https://user-images.githubusercontent.com/61971624/123266171-2726b180-d4d2-11eb-8912-d9cce5985a5f.png) | ![realizations real desktop](https://user-images.githubusercontent.com/61971624/123266523-7cfb5980-d4d2-11eb-801c-b54057e5cc04.png) |

| Mockup (Mobile inspiration) | Real (Mobile) |
|--------|------|
| ![inspiration realizations mobile](https://user-images.githubusercontent.com/61971624/123266710-a7e5ad80-d4d2-11eb-9438-11021015ffaf.png) | ![realizations real mobile](https://user-images.githubusercontent.com/61971624/123266978-e24f4a80-d4d2-11eb-9d0c-5996b1dbdf84.png) |

## How to test

1. Open the terminal and go to the folder of this project

2. In terminal:
   1. `git checkout develop`
   2. `git pull`
   3. `git checkout feature/create-realizations-view`
   4. `git pull origin feature/create-realizations-view`
   5. `npm install`
   6. `cd server/
   7. `node index.js`
   8. Open a new terminal
   9. `cd client/`
   10. `npm start`

3. In your browser go to http://localhost:3000 (should already be opened by `npm start`)

4. Check if the realizations view looks just like the mockup in different screen sizes, and in both the mobile and desktop versions. Check if slick (carousel) is working as it should and is fully functional in any screen size.

5. In your browser go to http://localhost:3001/admin/realizacoes

6. Check if you can create a new component and if the fields are similar to the ones required in the new mockup realizations version. Create  realizations components and see if they appear inside the carousel with its correct informations displaying. Make sure to also delete realizations nad make it less than 5 components, see if they are adjusted correctly in all screen sizes and if the carousel is working as it should.

7. After you are done, make sure to create more than 5 realizations cards, so it can be shown just like the mockup (sorry for the trouble :/ )

8. Read the changed files section on GitHub 

9. Test the things that you think are worth testing, even the ones that are not in this description
